src/gradualizer_cli.erl

```diff
@@ -48,7 +48,15 @@ print_usage() ->
     io:format("                                  - the default behaviour~n"),
     io:format("       --stop-on-first-error     stop type checking at the first error~n"),
     io:format("       --no-stop-on-first-error  inverse of --stop-on-first-error~n"),
-    io:format("                                  - the default behaviour~n").
+    io:format("                                  - the default behaviour~n"),
+    io:format("       --fmt-location            How to format location when pretty printing errors~n"),
+    io:format("                                 (Column is only available if analyzing from source)~n"),
+    io:format("                                 - 'none': no location for easier comparison~n"),
+    io:format("                                 - 'brief': for machine processing~n"),
+    io:format("                                   (\"LINE:COLUMN:\" before message text)~n"),
+    io:format("                                 - 'verbose': for human readers~n"),
+    io:format("                                   (\"on line LINE at column COLUMN\" within the message text)~n").
+
```

Comment:
It's be good to indicate which formatter is used by default if nothing is specified.

Reply:
true 👍